src/main/java/com/hazelcast/commandline/member/ProcessUtil.java

```diff
@@ -0,0 +1,127 @@
+package com.hazelcast.commandline.member;
+
+import com.hazelcast.commandline.member.names.MobyNames;
+import com.hazelcast.core.HazelcastException;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.hazelcast.commandline.HazelcastCommandLine.HAZELCAST_HOME;
+import static com.hazelcast.commandline.HazelcastCommandLine.SEPARATOR;
+
+public class ProcessUtil {
```

Comment:
1. `ProcessUtil` is not a good name. What you actually implement is a storage for processes. So, you could name it, for example, `ProcessStore`. You could then piggy back on the Spring Data repository naming and call methods `save()`, `delete()`, `find()`.
2. Make the class and all the methods package-private
3. Consider creating a stateful class from it then initialize an object of it in `HazelcastCommandLine`. Then you could actually add another constructor which takes the `instanceFilePath` and create a proper Unit Test for this class.

Reply:
Done 👍